### DISCUSSION TITLE
How do I change direction of stitch?

### DISCUSSION THREAD

**Discussion by jiriks74** [inkstitch/inkstitch#1073] [Q&A]
Hello, 
I'm learning InkStitch, and I just have to say, you guys are awesome. The documentation/tutorial videos are sometimes different to current version, so it can take quite a while to figure something out, but in the and I got it.

Now to my problem. I have a picture, which I vectorized and then I used `Break Apart Fill Objects`. I'm trying to change the direction of the stich by pressing `p`, holding `shift` and making a line over one of the objects. But when I do `Params` it is recognized as stitch, not as stitch direction. What am I doing wrong?

**Comment by Marif91** [comment]
Bonjour,
Le changement de direction des points de remplissages se fait dans la fenêtre des Paramètres... après avoir sélectionné chemin correspondant à la surface à remplir.
